src/coreclr/src/jit/optcse.cpp

```diff
@@ -401,10 +401,27 @@ void Compiler::optValnumCSE_Init()
 //
 unsigned Compiler::optValnumCSE_Index(GenTree* tree, Statement* stmt)
 {
-    unsigned key;
+    size_t   key;
     unsigned hash;
     unsigned hval;
     CSEdsc*  hashDsc;
+    bool     isIntConstHash       = false;
+    bool     enableSharedConstCSE = false;
+    int      configValue          = JitConfig.JitDisableConstCSE();
+
+#if defined(TARGET_ARM64)
+    // ARM64 - allow to combine with nearby offsets, when config is not 2
```

Comment:
should not 4 and 1 disable it as well for arm64?

Reply:
Yes, I will change that